index.html

```diff
@@ -1,16 +1,140 @@
+      <h1 class="mt-1 mb-3 red-color">Best Online Courses 2021</h1>
+      <p class="main-header__discreption p-2 mb-2 dark-gray">
+        Learn anything you want at any topics in Web, Mobile and Game development. There is also courses in other
+        fields.
+      </p>
+      <h3 class="mb-1 dark-gray">
+        Get professional skill
+      </h3>
+      <p class="dark-gray">
+        30 days money back guarantee and lifetime access
+      </p>
+    </header>
+    <section class="main-program bg-dark text-light pt-4 pb-5">
+      <h2 class="text-center underline mt-4">Main Program</h2>
+      <ul class="features mt-5 p-2 d-flex flex-md-row flex-column text-md-center gap-md-1 px-md-5">
+        <li class="feature d-flex flex-md-column align-items-center p-2 mb-3">
+          <i class="fas fa-chalkboard-teacher display-6 col-1 col-md-auto"></i>
+          <h3 class="h2 red-color col-4 col-md-auto">Lecture</h3>
+          <p class="h6">
+            Get courses from instructors that are experienced enough and have many years of experiences.
+          </p>
+        </li>
+        <li class="feature d-flex flex-md-column align-items-center p-2 mb-3">
+          <i class="far fa-folder display-6 col-1 col-md-auto"></i>
+          <h3 class="h2 red-color col-4 col-md-auto">Files</h3>
+          <p class="h6">
+            You can download all of the resources once you bought a course
+          </p>
+        </li>
+        <li class="feature d-flex flex-md-column align-items-center p-2 mb-3">
+          <i class="far fa-comment-dots display-6 col-1 col-md-auto"></i>
+          <h3 class="h2 red-color col-4 col-md-auto">Forum</h3>
+          <p class="h6">
+            If you have any problem don't hesitate to contact us and we fix the problem
+          </p>
+        </li>
+        <li class="feature d-flex flex-md-column align-items-center p-2 mb-3">
+          <i class="far fa-building display-6 col-1 col-md-auto"></i>
+          <h3 class="h2 red-color col-4 col-md-auto">Workshop</h3>
+          <p class="h6">
+            We have multiple workshops to test what you have learned
+          </p>
+        </li>
+        <li class="feature d-flex flex-md-column align-items-center p-2 mb-3">
+          <i class="fas fa-user-edit display-6 col-1 col-md-auto"></i>
+          <h3 class="h2 red-color col-4 col-md-auto">Review</h3>
+          <p class="h6">
+            You can review the course if you buy it or see all course reviews
+          </p>
+        </li>
+      </ul>
+      <a href="#" class="join-button">Join our academy</a>
+    </section>
+    <section class="featured-speakers pb-5">
+      <h2 class="text-center underline mt-5">Featured Instructors</h2>
+      <ul id="speakers"
+        class="speakers col-md-auto mt-5 d-flex flex-md-row flex-column px-md-3 flex-md-wrap">
+      </ul>
+      <a id="more-speakers-botton" class="more-botton d-md-none d-flex">
+        More
+        <div class="down-arrow"></div>
```

Comment:
- [ ] Articles in your mobile screen are a bit uneven in alignment and positioning on `mobile` screen, kindly investigate the issue.
Reference Design
![image](https://user-images.githubusercontent.com/33767951/142097123-0cfb6309-0a0d-428b-a5eb-dadb6fdbfc52.png)

Your Styling
![image](https://user-images.githubusercontent.com/33767951/142097085-b2386280-c113-4e2a-a777-7b58838db507.png)

Reply:
Done